modules/blocks.php

```diff
@@ -60,6 +66,37 @@ function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
 	}
 }
 
+/**
+ * Giphy block registration/dependency declaration.
+ *
+ * @param array $attr - Array containing the map block attributes.
+ *
+ * @return string
+ */
+function jetpack_giphy_block_load_assets( $attr ) {
+	$align       = isset( $attr['align'] ) ? $attr['align'] : 'center';
+	$style       = 'padding-top:' . $attr['paddingTop'];
+	$giphy_url   = isset( $attr['giphyUrl'] ) ? $attr['giphyUrl'] : '//giphy.com/embed/ZgTR3UQ9XAWDvqy9jv';
+	$search_text = isset( $attr['searchText'] ) ? $attr['searchText'] : '';
+	$caption     = isset( $attr['caption'] ) ? $attr['caption'] : null;
+
+	ob_start();
+	?>
+	<div class="wp-block-jetpack-giphy align<?php echo esc_attr( $align ); ?>">
+		<figure style="<?php echo esc_attr( $style ); ?>">
+			<iframe src="<?php echo esc_attr( $giphy_url ); ?>" title="<?php echo esc_attr( $search_text ); ?>"></iframe>
```

Comment:
Should we use `esc_url` instead of `esc_attr` here for the URL?

Reply:
Yup, resolved here: https://github.com/Automattic/jetpack/pull/10989/commits/71d7d458d4aac2c58b2446de4495bac5a94faa03